app/controllers/application_controller.rb

```diff
@@ -367,6 +367,7 @@ def report_data
     if options.nil? || options[:view].nil?
       model_view = process_params_model_view(params, options)
       @edit = session[:edit]
+      options_from_session(options)
```

Comment:
what about something like:
```
options.update(session[:paged_view_search_options])
```
would that work instead of `options_from_session`?